website/docs/deployment.md

```diff
@@ -32,7 +32,7 @@ You may refer to GitHub Pages' documentation [User, Organization, and Project Pa
 
 Example:
 
-```js
```

Comment:
Let's leave them as js. Why spell it out when it isn't necessary?

Reply:
js is not a valid language in prismjs.

only jsx and javascript works.
just found out today because actually our js syntax never really works properly, the prismjs is just trying its best to guess the language

see https://www.lucidar.me/en/web-dev/list-of-supported-languages-by-prism/

Reply:
so our choice is either “jsx” or “javascript”

Reply:
Oh god this is so subtle. Shall we include this in the docs now before we forget?

Reply:
in v1 i aliased js as jsx for prismjs so that ppl dont get confused.

we need custom remark/rehype prismjs plugin highlighting to do that
currently i use rehype/prismjs plugin from mapbox